sdk/core/azure-core/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ### Bug Fixes
 
 - Do not re-use a libcurl connection to same host but different port.
+- Ensure uniqueness of Uuid on posix platforms.
```

Comment:
```suggestion
- Ensure uniqueness of `Azure::Core::Uuid` on POSIX platforms.
```

Reply:
Namespacing is optional, but I think backticking and all caps POSIX should be there.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/pull/2208